html/modules/custom/reliefweb_reporting/src/Commands/ReliefWebReportingCommands.php

```diff
@@ -760,6 +776,52 @@ public function exportReportData(
       fclose($file);
     }
 
+    if ($upload) {
+      $client = new Client();
+      if ($credentials = getenv('GOOGLE_APPLICATION_CREDENTIALS')) {
+        // Suppress error to avoid echoing the credential in a traceback.
+        if (!@file_exists($credentials)) {
+          $client->setAuthConfig($credentials);
+        }
+        else {
+          $client->useApplicationDefaultCredentials();
+        }
+      }
+      else {
+        $this->logger->error('Error: No credentials defined in the GOOGLE_APPLICATION_CREDENTIALS environment.');
+        return FALSE;
+      }
+
+      $client->setApplicationName("Reliefweb Reports Data Uploader");
+      $client->setScopes(['https://www.googleapis.com/auth/drive']);
+      $service = new Drive($client);
+
+      $file = new DriveFile();
+      $file->setName(basename($output));
+      $file->setParents([$options['gdrive-upload-folder']]);
+
+      try {
+        $result = $service->files->create($file, [
+          'data' => file_get_contents($output),
+          'mimeType' => 'text/tab-separated-values',
+          'uploadType' => 'multipart',
```

Comment:
Google says to use `resumable ` as upload type for large files (> 5 MB): https://developers.google.com/drive/api/guides/manage-uploads?hl=en#php_1